mep/people/admin.py

```diff
@@ -436,8 +476,61 @@ class Media:
         ]
 
 
+
+
+class PersonResource(ModelResource):
+    def before_import(self, dataset, *args, **kwargs):
+        # lower and camel_case headers
+        dataset.headers = [x.lower().replace(' ','_') for x in dataset.headers]
+        # turn off indexing temporarily
+        IndexableSignalHandler.disconnect()
+        ViafSignalHandler.disconnect()
+
+    def after_import(self, *args, **kwargs):
+        super().after_import(*args, **kwargs)
+        # reconnect indexing signal handler
+        IndexableSignalHandler.connect()
+        ViafSignalHandler.connect()
+
+    def before_import_row(self, row, **kwargs):
+        # gender to one char
+        def fmt_gender(x):
+            x = str(x).strip()
+            return x[0].upper() if x else ''
+        row['gender']=fmt_gender(row.get('gender'))
+
+        def fmt_nation(x):
+            x=str(x).strip()
+            if not x or x=='[no country]': return
+            return x
+        row['nation'] = fmt_nation(row.get('nation'))
```

Comment:
does the natural foreign key option get us anything here?  (i.e., could we use that and get rid of the custom format could?)

https://django-import-export.readthedocs.io/en/latest/api_resources.html#import_export.resources.ResourceOptions.use_natural_foreign_keys

It looks like we haven't implemented that for `Country`, but it isn't hard to add and would be valuable for other things.